envoy/api/v2/cds.proto

```diff
@@ -356,6 +356,11 @@ message Cluster {
     // .. note::
     //   The specified percent will be truncated to the nearest 1%.
     core.Percent healthy_panic_threshold = 1;
+    //Configuration for ZoneAwareLoadBalancer
+    // [#not-implemented-hide:] If specified envoy can use this config option to decide on applying ZoneAwareLoadBalancing
```

Comment:
Please wrap to ~100 cols. Same below. Each option should document what the defaults are if not set. Each option should also ref-link to the existing docs on zone aware load balancing as well as the runtime values.

Reply:
Thanks for the review . Fixed with new push.